### PR TITLE
Update troubleshooting document for Cron service info

### PR DESCRIPTION
**Pull request by zjalali**

### CODE REVIEW

File: docs/Troubleshooting.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
 * OMI package was manually upgraded to a newer version compared to what was installed by OMS Agent package
 * DSC resource logs "class not found" error in omsconfig.log log file
 * OMS Agent for Linux data is backed up
+* Cron service is not running
```
Comment (NarineM) [inline docs/Troubleshooting.md]: Hi, can you change this line to be the dsc issue that impacts the customer when cron job is not running?

File: docs/Troubleshooting.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
...
 * In some cases, when the OMS Agent for Linux cannot talk to the OMS Service, data on the Agent is backed up to the full buffer size: 50 MB. The OMS Agent for Linux should be restarted by running the following command `/opt/microsoft/omsagent/bin/service_control restart`.
  * **Note:** This issue is fixed in Agent version >= 1.1.0-28
-* Make sure the _cron_ service is running. You can use `service cron status` (Debian, Ubuntu, SUSE) or `service crond status` (RHEL, CentOS) to check the status of this service. If the service does not exist, you can install the binaries and start the service using the following:
+* If `omsconfig.log` log file does not indicate that `PerformRequiredConfigurationChecks` operations are running periodically on the system, there might be a problem with the _cron_ job/service. Make sure _cron_ job exists under `/etc/cron.d/OMSConsistencyInvoker`. Also, make sure the _cron_ service is running. You can use `service cron status` (Debian, Ubuntu, SUSE) or `service crond status` (RHEL, CentOS) to check the status of this service. If the service does not exist, you can install the binaries and start the service using the following:
```
Comment (NarineM) [inline docs/Troubleshooting.md]: Looks good, except I see _ (underscore before and after the work cron can you remove that/

Comment (zjalali) [reply docs/Troubleshooting.md]: That makes it _Italic_ and it won't show up in the actual document, but I'll remove it.

**Comment by NarineM** [comment]
hi, when updating PR please either ruse --emend option or squash the commits so that they appear as a single commit.

File: docs/Troubleshooting.md [hunk 3]
```diff
@@ -215,8 +215,9 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
 * Connection to the OMS Service is blocked
 * VM was rebooted
 * OMI package was manually upgraded to a newer version compared to what was installed by OMS Agent package
-* DSC resource logs "class not found" error in omsconfig.log log file
+* DSC resource logs "class not found" error in `omsconfig.log` log file
 * OMS Agent for Linux data is backed up
+* DSC resouce logs "Current configuration does not exist. Execute Start-DscConfiguration command with -Path parameter to specify a configuration file and create a current configuration first." in `omsconfig.log` log file, but no log message exists about `PerformRequiredConfigurationChecks` operations.
```
Comment (NarineM) [inline docs/Troubleshooting.md]: one last thing, it is not dsc resource that logs current configuration does not exist, but the LCM or the engine , I think.

Comment (NarineM) [reply docs/Troubleshooting.md]: you can change to just be 'DSC logs ...."Current..." .."

**Pull request merged**